docker/backend-dev/conf/Config2.pm

```diff
@@ -39,23 +42,30 @@ BEGIN
 		$memd_servers
 		$facebook_app_id
 	    $facebook_app_secret
+		$google_cloud_vision_api_key
+		$crowdin_project_identifier
+		$crowdin_project_key
+		$csrf_secret
 		$robotoff_url
 		%server_options
 	);
 	%EXPORT_TAGS = (all => [@EXPORT_OK]);
 }
 use vars @EXPORT_OK ; # no 'my' keyword for these
+use strict;
+use utf8;
 
 # server constants
-$server_domain = "productopener.localhost";
+$server_domain = $ENV{PRODUCT_OPENER_DOMAIN};
 
-@ssl_subdomains = qw();
+@ssl_subdomains = qw(
+*
```

Comment:
This doesn't work with the simple `start_dev.bat` script, or when you start it from VS Code as a dev container, because we currently don't have TLS set up for local development. We could either tell nginx to serve HTTPS for local dev, or make this configurable via `$ENV`, too.